internal/domain/meter/model.go

```diff
@@ -9,12 +9,20 @@ import (
 )
 
 type Meter struct {
-	ID          string      `db:"id" json:"id"`
-	EventName   string      `db:"event_name" json:"event_name"`
-	Aggregation Aggregation `db:"aggregation" json:"aggregation"`
+	ID          string           `db:"id" json:"id"`
+	EventName   string           `db:"event_name" json:"event_name"`
+	Name        string           `db:"name" json:"name"`
+	Aggregation Aggregation      `db:"aggregation" json:"aggregation"`
+	Filters     []Filter         `db:"filters" json:"filters"`
```

Comment:
Will have to give a repo and API for update meter so that users can update the filter values